common/deploy.sh

```diff
@@ -22,10 +22,14 @@ check_rancher_vars
 
 echo "Deploying rancher stack $RANCHER_STACK_NAME in environment $RANCHER_ENVIRONMENT"
 
+
+# Set RANCHER_URL, doing it here so we can set RANCHER_SERVER in Vault
+echo "RANCHER_URL=https://${RANCHER_SERVER}/v2-beta/schemas"
```

Comment:
RANCHER_URL is checked for in check_rancher_vars(). Not sure if that is still a valid thing to do.

Reply:
@neilbmclaughlin I've removed from the function now.